src/lib.rs

```diff
@@ -211,9 +211,38 @@ mod tests {
         pub utxos: Vec<Utxo>,
     }
 
+    // TODO check about adding this to rust-bitcoins from_str for Weight
+    fn parse_weight(weight: &str) -> Weight {
+        let size_parts: Vec<_> = weight.split(" ").collect();
+        let size_int = size_parts[0].parse::<u64>().unwrap();
+        match size_parts[1] {
+            "wu" => Weight::from_wu(size_int),
+            "vb" => Weight::from_vb(size_int).unwrap(),
+            _ => panic!("only support wu or vb sizes"),
```

Comment:
Nit: Please feel free to ignore. "b" is the symbol for bit, and "B" is the symbol for byte, so a virtualbyte should have a capital B. (I use "vB".)

Reply:
Oops thanks, I'll update this.

Reply:
update here https://github.com/p2pderivatives/rust-bitcoin-coin-selection/pull/94